### ISSUE TITLE
Need to call addSimpleTouchListener() after implementing touch events

### ISSUE THREAD

**Issue by horaciocome1** [horaciocome1/simple-recyclerview-touch-listener#1]
it is kinda akward having to call two funtions to perform only one action
```kotlin
recyclerView.setOnClick {}
recyclerView.addSimpleTouchListener()
```

**Comment by horaciocome1** [comment]
It also makes it easy to forget and without it the implementation simple dont run

**Comment by horaciocome1** [comment]
solved on release 0.2.0
